code/modules/vehicles/mecha/mecha_construction_paths.dm

```diff
@@ -417,12 +418,12 @@
 			"key" = /obj/item/stack/rods,
 			"amount" = 10,
 			"back_key" = TOOL_WELDER,
-			"desc" = "Outer Plating is welded."
+			"desc" = "Outer plating is welded, and 10 <b>rods</b> can be used to install the cockpit wire."
```

Comment:
```suggestion
			"desc" = "Outer plating is welded, and 10 <b>rods</b> can be used to install the cockpit."
```
Mostly nitpick, and if you can think of another word to replace it with that's fine too, since we use the word `wiring` in mech construction to mean cable coils.